# This file is setting up all the settings and options for the robot.

# Control ROS loop rate
spin_rate: 20  
#Not used

# Gripper pressure in psi
gripper_pressure: 5.0

# PCA9685 Oscillator Frequency
# this value must be calibrated for the specific device
oscillator_freq: 26000000

# Polhemus Settings
soft_arm_tip_station: 0
##rigid_arm_tip_station: 1

# Stepper Values
# stepper_offset: 1.0

# Correction
theta_1_correction: 0.0
theta_2_correction: 0.0
theta_3_correction: 0.0
theta_4_correction: 0.0

# Large pan correction allows for good range of motion, but
# the zero is aligned with an axis
pan_correction: 0 #49.0
tilt_correction: 0

# DAC I2C Addresses
dac_address: 0x62
# 0x62 is adafruit default
# 0x63 is adafruit A0 set to VCC
# 0x60 is sparkfun - needs different wiring
# 0x61 is sparkfun - needs different wiring
